py/selenium/webdriver/common/action_chains.py

```diff
@@ -274,18 +276,8 @@ def send_keys_to_element(self, element, *keys_to_send):
         return self
 
     def _keys_to_typing(self, value):
-        typing = []
-        for val in value:
-            if isinstance(val, Keys):
-                typing.append(val)
-            elif isinstance(val, int):
-                val = str(val)
-                for i in range(len(val)):
-                    typing.append(val[i])
-            else:
-                for i in range(len(val)):
-                    typing.append(val[i])
-        return typing
+        """DEPRECATED: Wrapper here just in case someone has been using this."""
```

Comment:
This is a private method, don't need to deprecate, just remove. We'll make mention in the changelog.